### PR TITLE
Create full_description.txt

### PR DESCRIPTION
**Pull request by daywalk3r666**

### CODE REVIEW

File: fastlane/metadata/android/de/full_description.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Dies ist eine inoffizielle Android-App für die DeepL-Übersetzung. Es verwendet eine Technologie namens WebView, eine Android-GUI-Komponente, um es wie eine native App aussehen zu lassen. JavaScript wird verwendet, um alle bis auf die notwendigsten Elemente auszublenden.
```
Comment (fm-sys) [inline fastlane/metadata/android/de/full_description.txt:1]:
```suggestion
Dies ist eine inoffizielle Android-App für den DeepL-Übersetzer. Es verwendet eine Technologie namens WebView, eine Android-GUI-Komponente, um es wie eine native App aussehen zu lassen. JavaScript wird verwendet, um alle bis auf die notwendigsten Elemente auszublenden.
```

File: fastlane/metadata/android/de/full_description.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+<b>Funktionen:</b>
+
+* Nur den Übersetzungsbildschirm von WebView anzeigen
```
Comment (fm-sys) [inline fastlane/metadata/android/de/full_description.txt:5]:
```suggestion
* Den Übersetzungsbildschirm im WebView anzeigen lassen 
```

**Review by fm-sys — COMMENTED** [review]
Here we go. I made a small review for my native language "German" as already in  #65...

File: fastlane/metadata/android/de/full_description.txt (hunk 2 above)
Comment (daywalk3r666) [reply fastlane/metadata/android/de/full_description.txt]: thx again. i edit the pull request

Comment (daywalk3r666) [reply fastlane/metadata/android/de/full_description.txt]: @fm-sys can you check my readme updates?

File: README_DE.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+![image](https://user-images.githubusercontent.com/53967490/89320092-fe2fdf00-d6bb-11ea-97d6-84fd66f73395.png)
+
+## Funktionen
+
+- Den Übersetzungsbildschirm im WebView anzeigen lassen.
+- In den Dunkelmodus wechseln.
+- Starten Sie DeepL über die schwebende Textauswahlaktion.
+
+## Anmerkung
+Dieses Projekt wurde mit DeepL übersetzt. Wenn es irgendwelche Übersetzungsfehler gibt, senden Sie bitte eine Anfrage!
```
Comment (fm-sys) [inline README_DE.md:22]:
```suggestion
Dieses Projekt wurde mit DeepL übersetzt. Sollte es irgendwelche Übersetzungsfehler geben, erstellen Sie gerne einen Pull Request!
```

There are no real translations for git commands. To make it more understandable, we should probably stay with "Pull Request".

That being said, LGTM 👍

**Comment by sakusaku3939** [comment]
Thanks a lot! 👍👍

**Pull request merged**